src/Mono.Android/Test/Java.Interop/JnienvTest.cs

```diff
@@ -453,7 +454,9 @@ public ContainsExportedMethods ()
 			Constructed = true;
 		}
 
+#if !NET // https://github.com/xamarin/xamarin-android/issues/5265
 		[Export]
```

Comment:
I think it should be enough to disable the `CreateTypeWithExportedMethods`  test on dotnet?

Reply:
nm. The build error is happening even when `Mono.Android.Export` is referenced. I will look at how to fix #5265